spaces/S000064/properties/P000118.md

```diff
@@ -0,0 +1,12 @@
+---
+space: S000064
+property: P000118
+value: true
+---
+
+A $k$-network can be constructed as the union of the following locally finite families:
+- $\{ \{x\}:x\in X\setminus D\}$ - the singletons of all points outside $D$, 
+- countably many Euclidean balls in $D$ forming a base for the standard topology on $\mathbb Q^2$ (each can be a lone member of a separate family),
+- $\{\{q\}\}$ for all $q\in D$.
```

Comment:
```suggestion
A $k$-network can be constructed as the union of the following countably many locally finite families:
- the family $\{ \{x\}:x\in X\setminus D\}$ of singletons of all points outside $D$, 
- a family $\{B\}$ for each Euclidean ball $B$ in $D$ belonging to a countable base for the standard topology on  $\mathbb Q^2$,
- a family $\{\{q\}\}$ for each $q\in D$.
```
making the description of the three types of families more parallel to each other